network/vpc/api-cli/vpc-private-beta.mdx

```diff
@@ -1,30 +1,29 @@
 ---
 meta:
-  title: VPC Regional Private Networks - Private Beta Guide
-  description: This page shows you how to get started with Scaleway VPC Regional Private Networks, if you are one of the Private Beta testers.
+  title: Regional Private Networks with the Scaleway API
+  description: This page shows you how to get started with regional Private Networks via the Scaleway API.
 content:
-  h1: VPC Regional Private Networks - Private Beta Guide
-  paragraph: This page shows you how to get started with Scaleway VPC Regional Private Networks, if you are one of the Private Beta testers.
+  h1: Regional Private Networks with the Scaleway API
+  paragraph: This page shows you how to get started with regional Private Networks via the Scaleway API.
 dates:
-  validation: 2023-03-07
-  posted: 2021-02-14
+  validation: 2023-03-22
+  posted: 2023-02-14
 categories:
   - network
 ---
 
-VPC Regional Private Networks allow you to create Private Networks that are scoped to a region (Paris, Amsterdam or Warsaw), rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
+Regional Private Networks allow you to create Private Networks that are scoped to a region (Paris, Amsterdam or Warsaw), rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
```

Comment:
```suggestion
Regional Private Networks allow you to create Private Networks that are scoped to a region rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
```